alerts/kube_apiserver.libsonnet

```diff
@@ -145,6 +145,31 @@ local utils = import 'utils.libsonnet';
               message: 'A client certificate used to authenticate to the apiserver is expiring in less than %s.' % (utils.humanizeSeconds($._config.certExpirationCriticalSeconds)),
             },
           },
+          {
+            alert: 'AggregatedAPIErrors',
+            expr: |||
+              sum by(name, namespace)(increase(aggregator_unavailable_apiservice_count[5m])) > 2
```

Comment:
My feeling is that this might be a bit brittle without a for clause, but at the same time I would also like to try it this way and see whether it's maybe not, and we can still add a for clause should it be too flaky.